### PR TITLE
br: add restore to systables

### PR DESCRIPTION
**Pull request by YuJuncen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a document about the ability of backing up / restore system tables provided by https://github.com/pingcap/br/pull/1048.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

Please hold for v5.0 and v4.0 until https://github.com/pingcap/br/pull/1048 was bring to those versions.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -377,6 +377,31 @@ br restore full \
 
 增量恢复的方法和使用 BR 进行全量恢复的方法并无差别。需要注意，恢复增量数据的时候，需要保证备份时指定的 `last backup ts` 之前备份的数据已经全部恢复到目标集群。
 
+### 恢复创建在 `mysql` 数据库下的表
+
+在 [#1048](https://github.com/pingcap/br/pull/1048) 之后，BR 可以并且默认会备份 `mysql` 数据库下的表。
```
Comment (SunRunAway) [inline br/use-br-command-line-tool.md:382]: 这个是对用户的文档，最好用版本而不是PR。
eg:
`在 4.0.X 和 5.0.X 之后`

Comment (YuJuncen) [reply br/use-br-command-line-tool.md:382]: 嗯，因为这个文档是 dev 的，所以好像只能这么写……在 cherry-pick 的时候应该要改动到对应版本……

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -377,6 +377,31 @@ br restore full \
...
+在恢复时，`mysql` 下的表默认会被 [table filter](/table-filter.md#表库过滤语法) 规则过滤掉，要恢复在 `mysql` 下的用户表的时候，可以通过 `-f` 来指定目标，例如，以下命令会恢复 `mysql.usertable`。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br restore full -f '*.*' -f '!mysql.*' -f 'mysql.usertable' -s $external_storage_url
+#               ^        ^             ^
+#               |        |             +- select `mysql`.`usertable`
+#               |        +- filter out other tables in `mysql`
+#               +- select all tables
```
Comment (kennytm) [inline br/use-br-command-line-tool.md:393]: use chinese for these 3 comments 😓

Comment (YuJuncen) [reply br/use-br-command-line-tool.md]: Done😂

**Comment by kennytm** [comment]
/lgtm

https://github.com/pingcap/docs-cn/pull/6194#pullrequestreview-648892549 should be addressed before merging.

File: br/use-br-command-line-tool.md (hunk 1 above)
Comment (TomShawn) [reply br/use-br-command-line-tool.md:382]: 在 dev 版文档的话，"... 之后"也可以去掉，因为用户看到的就是最新开发版文档。
```suggestion
BR 可以并且会默认备份 `mysql` 数据库下的表。
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -377,6 +377,31 @@ br restore full \
 
 增量恢复的方法和使用 BR 进行全量恢复的方法并无差别。需要注意，恢复增量数据的时候，需要保证备份时指定的 `last backup ts` 之前备份的数据已经全部恢复到目标集群。
 
+### 恢复创建在 `mysql` 数据库下的表
+
+在 [#1048](https://github.com/pingcap/br/pull/1048) 之后，BR 可以并且默认会备份 `mysql` 数据库下的表。
+
+在恢复时，`mysql` 下的表默认会被 [table filter](/table-filter.md#表库过滤语法) 规则过滤掉，要恢复在 `mysql` 下的用户表的时候，可以通过 `-f` 来指定目标，例如，以下命令会恢复 `mysql.usertable`。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:384]:
```suggestion
在恢复时，`mysql` 下的表默认会被 [table filter](/table-filter.md#表库过滤语法) 规则过滤掉。如果需要恢复 `mysql` 下的用户表，可以通过 `-f` 来指定目标用户表。以下示例中的要恢复目标用户表为`mysql.usertable`。
```

File: br/use-br-command-line-tool.md [hunk 4]
```diff
@@ -377,6 +377,31 @@ br restore full \
...
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br restore full -f '*.*' -f '!mysql.*' -f 'mysql.usertable' -s $external_storage_url
+#               ^        ^             ^
+#               |        |             +-  ……仅恢复 `mysql`.`usertable`。
+#               |        +- ……但是过滤掉 `mysql` 中的其它表……
+#               +- 恢复所有表……
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:393]: 能否将注释拿出来单独描述？这里使用了三个 -f，如果不能在一条命令中同时指定，是否需要写三个命令来分开说明呢？

File: br/use-br-command-line-tool.md [hunk 5]
```diff
@@ -377,6 +377,31 @@ br restore full \
...
+```
+
+> **警告：**
+>
+> 即便系统表（例如，`mysql.tidb` 等）也可以通过这个功能备份和恢复，但是部分系统表在恢复之后可能会出现意外状况，已知的异常如下：
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:398]:
```suggestion
> 虽然系统表（例如 `mysql.tidb` 等）可以通过 BR 进行备份和恢复，但是部分系统表在恢复之后可能会出现意外状况，已知的异常如下：
```

File: br/use-br-command-line-tool.md [hunk 6]
```diff
@@ -377,6 +377,31 @@ br restore full \
...
+>
+> - 统计信息表（`mysql.stat_*`）无法被恢复。
+> - 用户信息表（`mysql.user`）在恢复之后不会生效，直到用户手动执行 `FLUSH PRIVILEGES`。
+> 
+> 对更多系统表恢复的兼容性测试也尚在进行中，为了防止意外，请尽量避免在生产环境中恢复系统表。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:403]:
```suggestion
> 恢复系统表可能还存在更多兼容性问题。为了防止意外发生，请避免在生产环境中恢复系统表。
```

File: br/use-br-command-line-tool.md [hunk 7]
```diff
@@ -377,6 +377,31 @@ br restore full \
 
 增量恢复的方法和使用 BR 进行全量恢复的方法并无差别。需要注意，恢复增量数据的时候，需要保证备份时指定的 `last backup ts` 之前备份的数据已经全部恢复到目标集群。
 
+### 恢复创建在 `mysql` 数据库下的表
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:380]: 这里是否需要加上"实验性功能"的说明？
```suggestion
### 恢复创建在 `mysql` 数据库下的表（实验性功能）
```

File: br/use-br-command-line-tool.md [hunk 8]
```diff
@@ -377,6 +377,31 @@ br restore full \
 
 增量恢复的方法和使用 BR 进行全量恢复的方法并无差别。需要注意，恢复增量数据的时候，需要保证备份时指定的 `last backup ts` 之前备份的数据已经全部恢复到目标集群。
 
+### 恢复创建在 `mysql` 数据库下的表（实验性功能）
+
+BR 可以并且会默认备份 `mysql` 数据库下的表。
+
+在恢复时，`mysql` 下的表默认会被 [table filter](/table-filter.md#表库过滤语法) 规则过滤掉。如果需要恢复 `mysql` 下的用户表，可以通过 `-f` 来指定目标用户表。以下示例中的要恢复目标用户表为`mysql.usertable`。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:384]:
```suggestion
在恢复时，`mysql` 下的表默认会被 [table filter](/table-filter.md#表库过滤语法) 规则过滤掉。如果需要恢复 `mysql` 下的用户表，可以添加 `-f` 来指定目标用户表。以下示例中的要恢复目标用户表为`mysql.usertable`。
```

File: br/use-br-command-line-tool.md [hunk 9]
```diff
@@ -377,6 +377,31 @@ br restore full \
...
+br restore full -f '*.*' -f '!mysql.*' -f 'mysql.usertable' -s $external_storage_url
+#               ^        ^             ^
+#               |        |             +-  ……仅恢复 `mysql`.`usertable`。
+#               |        +- ……但是过滤掉 `mysql` 中的其它表……
+#               +- 恢复所有表……
+```
+
+> **警告：**
+>
+> 虽然系统表（例如 `mysql.tidb` 等）可以通过 BR 进行备份和恢复，但是部分系统表在恢复之后可能会出现意外状况，已知的异常如下：
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:398]:
```suggestion
> 虽然系统表（例如 `mysql.tidb` 等）可以通过 BR 进行备份和恢复，但是部分系统表在恢复之后可能会出现非预期的状况，已知的异常如下：
```

File: br/use-br-command-line-tool.md (hunk 4 above)
Comment (TomShawn) [reply br/use-br-command-line-tool.md:393]: @YuJuncen PTAL

Comment (TomShawn) [reply br/use-br-command-line-tool.md:393]: @3pointer PTAL, thanks!

Comment (YuJuncen) [reply br/use-br-command-line-tool.md:393]: 这里的三个 -f 应该是要在一条命令中同时指定的，稍后我处理一下这个……

File: br/use-br-command-line-tool.md [hunk 10]
```diff
@@ -377,6 +377,39 @@ br restore full \
...
+
+在执行恢复时，`mysql` 下的表默认不会被恢复。如果需要恢复 `mysql` 下的用户创建的表，可以通过 [table filter](/table-filter.md#表库过滤语法) 来显式地包含目标表。以下示例中要恢复目标用户表为 `mysql.usertable`；该命令会在执行正常的恢复的同时恢复 `mysql.usertable`。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+br restore full -f '*.*' -f '!mysql.*' -f 'mysql.usertable' -s $external_storage_url
+```
+
+在如上的命令中，使用 `-f '*.*'` 来覆盖掉默认的规则，然后通过 `-f '!mysql.*'` 来命令 BR 不要恢复 `mysql` 中的表，除非另有指定。最后的 `-f 'mysql.usertable'` 则指定需要恢复 `mysql.usertable`。具体原理请参考 [table filter 的文档](/table-filter.md#表库过滤语法)。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:392]:
```suggestion
在如上的命令中，`-f '*.*'` 用于覆盖掉默认的规则，`-f '!mysql.*'` 指示 BR 不要恢复 `mysql` 中的表，除非另有指定。`-f 'mysql.usertable'` 则指定需要恢复 `mysql.usertable`。具体原理请参考 [table filter 的文档](/table-filter.md#表库过滤语法)。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**